comp/core/tagger/component.go

```diff
@@ -18,20 +17,23 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/tagger/collectors"
 	"github.com/DataDog/datadog-agent/pkg/tagger/types"
 	"github.com/DataDog/datadog-agent/pkg/tagset"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+	"go.uber.org/fx"
 )
 
-// Tagger is an interface for transparent access to both localTagger and
-// remoteTagger.
-type Tagger interface {
-	Init(context.Context) error
+type Component interface {
+	Start(ctx context.Context) error
```

Comment:
Note: In a later PR, `Start` method should be removed in favour of fx.Lifecyle.

Reply:
I will do it in the following PR. It can be moved to an internal helper function that is executed during fx OnStart.

Reply:
could you add a comment in this PR to not forget? 
```suggestion
    // TODO: Start method should be removed in favour of fx.Lifecyle.
	Start(ctx context.Context) error
```